<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditMember</class>
 <widget class="QDialog" name="EditMember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">color:rgb(111, 76, 19);
font: 18pt &quot;Segoe UI&quot;;</string>
     </property>
     <property name="title">
      <string>Edit member details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_idm">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Member ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_mID">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_name">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_name</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_name">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;
</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_phone">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Phone number</string>
            </property>
            <property name="buddy">
             <cstring>lineEdit_pnum</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_pnum">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;
</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_sub">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="text">
             <string>Date of subscription</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_subdate">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;

</string>
            </property>
            <property name="placeholderText">
             <string>YYYY-MM-DD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Subscription end date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_enddate">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 12pt &quot;Segoe UI&quot;;
</string>
            </property>
            <property name="placeholderText">
             <string>YYYY-MM-DD</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">
font: 12pt &quot;Segoe UI&quot;;
background: transparent;
border: none;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>25</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cancel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">color:rgb(111, 76, 19);
font: 14pt &quot;Segoe UI&quot;;</string>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
